lib/gemeinsam_sport_frontend_web/templates/room/room.html.eex

```diff
@@ -57,6 +57,38 @@ document.body.onload = function() {
     const options = {
         roomName: roomId,
         parentNode: document.querySelector('#meet'),
+        interfaceConfigOverwrite: {
+            TOOLBAR_BUTTONS: [
+                'microphone',
+                'camera',
+                //'closedcaptions',
+                //'desktop',
+                'fullscreen',
+                //'fodeviceselection',
+                'hangup',
+                //'profile',
+                //'info',
```

Comment:
I think you can leave out the disabled ones... If we keep them for reference we can never be sure whether the list is still up-to-date. But I do not mind too much...

Reply:
Replaced it with a comment to the source